curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md

```diff
@@ -20,13 +20,13 @@ assert(document.querySelectorAll('a').length === 1);
 The `p` element should still have the anchor (`a`) element inside of it.
 
 ```js
-assert($('p > a').length);
+assert(document.querySelectorAll('p > a').length);
 ```
 
 The link's `href` value should be `https://freecatphotoapp.com`. You have either omitted the `href` value or have a typo.
 
 ```js
-const nestedAnchor = $('p > a')[0];
+const nestedAnchor = document.querySelectorAll('p > a');
```

Comment:
`querySelectorAll` returns a `NodeList` if you just want a single element use `querySelector`.

Reply:
Ahh! just the autocomplete playing games with me. I will change it.